torchani/models.py

```diff
@@ -1,37 +1,50 @@
 # -*- coding: utf-8 -*-
 """The ANI model zoo that stores public ANI models.
 
-Currently the model zoo has two models: ANI-1x and ANI-1ccx. The corresponding
-classes of these two models are :class:`ANI1x` and :class:`ANI1ccx`. These
-classes share the same API. To use the builtin models, you simply need to
-create an object of its corresponding class. These classes are subclasses of
-:class:`torch.nn.Module`, and could be used directly. Below is an example of
-how to use these models:
+Currently the model zoo has two models: ANI-1x and ANI-1ccx. The classes
+of these two models are :class:`ANI1x` and :class:`ANI1ccx`,
+these are subclasses of :class:`torch.nn.Module`.
+To use the models just instantiate them and either
+directly calculate energies or get an ASE calculator. For example:
 
 .. code:: python
-
-    model = torchani.models.ANI1x()
+    ani1x = torchani.models.ANI1x()
     # compute energy using ANI-1x model ensemble
-    _, energies = model((species, coordinates))
-    model.ase()  # get ASE Calculator using this ensemble
+    _, energies = ani1x((species, coordinates))
+    ani1x.ase()  # get ASE Calculator using this ensemble
     # convert atom species from string to long tensor
-    model.species_to_tensor('CHHHH')
+    ani1x.species_to_tensor('CHHHH')
 
-    model0 = model[0]  # get the first model in the ensemble
+    model0 = ani1x.neural_networks[0]  # get the first model in the ensemble
```

Comment:
This should be `ani1x[0]`. The difference is `ani1x.neural_networks[0]` only get the neural network, but `ani1x[0]` would obtain a `Sequential` of aev computer, and neural network, and energy shifter, and it also has `.ase()` to get ase calculator.

Reply:
Fixed in a3882f0